site-src/concepts/use-cases.md

```diff
@@ -0,0 +1,177 @@
+rollout](/guides/traffic-splitting), etc., while not worrying about the things
+that Charlie and Ian are handling.
+
+[HTTPRoute]:/api-types/httproute
+[GatewayClass]:/api-types/gatewayclass
+[Gateway]:/api-types/gateway
+[Route Attachment Process]:/concepts/api-overview#attaching-routes-to-gateways
+
+![Gateway API Roles](/images/gateway-roles.png)
+
+## Basic [east/west] use case
+
+In this scenario, Ana has built a workload which is already running in a
+cluster with a [GAMMA]-compliant [service mesh]. She wants to use the mesh to
+protect her workload by rejecting calls to her workload with incorrect
+URL paths, and by enforcing timeouts whenever anyone makes a request of her
+workload.
+
+- Charlie and Ian have already provided a cluster with a running service mesh.
+  Ana doesn't need to make any requests of them.
+
+- Ana writes an HTTPRoute that defines acceptable routes and timeouts and has
+  a `parentRef` of her workload's Service.
+
+- Ana applies her HTTPRoute in the same Namespace as her workload.
+
+- The mesh automatically starts enforcing the routing policy described by
+  Ana's HTTPRoute.
+
+In this case, the separation of concerns across roles allows Ana to take
+advantage of the service mesh, with custom routing logic, without any
+bottlenecks in requests to Charlie or Ian.
+
+[east/west]:/concepts/glossary#east-west-traffic
+[GAMMA]:/contributing/gamma/
+[service mesh]:/concepts/glossary#service-mesh
+
+## Gateway and mesh use case
+
+This is effectively a combination of the `multiple applications behind a
+single Gateway](#multiple-applications-behind-a-single-gateway) and [basic
+east/west](#basic-east-west-use-case) use cases:
+
+- Charlie and Ian will provision a cluster, a [GatewayClass], and a [Gateway].
+
+- Ana and Allison will deploy their applications in the appropriate
+  Namespaces.
+
+- Ana and Allison will then apply HTTPRoute resources as appropriate.
+
+There are two very important changes in this scenario, though, since a mesh is
+involved:
+
+1. If Charlie has deployed a [gateway controller] that defaults to [Service
+   routing], they will probably need to reconfigure it for [endpoint routing].
+   (This is an ongoing area of work for [GAMMA], but the expectation is that
+   endpoint routing will be recommended.)
+
+2. Ana and/or Allison will need to bind HTTPRoutes to their respective
+   workloads' Services to configure mesh routing logic. These could be
+   distinct HTTPRoutes solely for the mesh, or they could apply single
+   HTTPRoutes that bind to both the Gateway and a Service.
+
+As always, the ultimate point of separating concerns in this way is that it
+permits Charlie to enforce centralized policies [such as
+TLS](/guides/tls#downstream-tls) at the Gateway, while allowing Ana and
+Allison to retain independent control of [routing
+logic](/guides/http-routing), [traffic splitting
+rollout](/guides/traffic-splitting), etc., both for [north/south] and for
+[east/west] routing.
```

Comment:
I think this section is great, but maybe it needs a larger callout about the details being a bit in flux? I don't want a repeat of Policy Attachment, where we make things seem like less experimental than they are by accident.

Reply:
Yeah really agree that we should be especially careful to add "experimental" notices/banners above all mesh-related sections (something like https://github.com/kubernetes-sigs/gateway-api/blob/main/site-src/api-types/grpcroute.md?plain=1#L3).

Reply:
Let me know if 41326713f314fed0abba6e759def365aceaa5f69 hits the mark here.

Reply:
@youngnick, I'm going to go ahead and resolve this, yell if you want further edits!